### PR TITLE
Content Security Policy

### LINKED ISSUE #270: Add CSP

### PR DESCRIPTION
**Pull request by cadecairos**
fixes #270

### CODE REVIEW

File: web/server.js [hunk 1]
```diff
@@ -32,7 +32,41 @@ module.exports = function(options) {
...
+          'data:',
+          'https://www.google-analytics.com',
+          'http://www.google-analytics.com'
+        ],
+        childSrc: [
+          '\'none\''
+        ],
+        scriptSrc: [
+          '\'self\'',
+          '\'unsafe-eval\'',
```
Comment (jbuck) [inline web/server.js]: :weary: 

What requires this?

Comment (cadecairos) [inline web/server.js]: Joi D:

File: web/server.js [hunk 2]
```diff
@@ -32,7 +32,41 @@ module.exports = function(options) {
...
+          '\'self\'',
+          'https://fonts.googleapis.com'
+        ],
+        imgSrc: [
+          '\'self\'',
+          'data:',
+          'https://www.google-analytics.com',
+          'http://www.google-analytics.com'
+        ],
+        childSrc: [
```
Comment (jbuck) [inline web/server.js]: Hm, any reason to call this out as `'non'`?

File: web/server.js [hunk 3]
```diff
@@ -32,7 +32,41 @@ module.exports = function(options) {
...
 
-  server.register(require('hapi-auth-cookie'), function(err) {
+  server.register([
+    require('hapi-auth-cookie'),
+    require('scooter'),
+    {
+      register: require('blankie'),
+      options: {
+        defaultSrc: [
+          '\'self\''
```
Comment (jbuck) [inline web/server.js]: Could we use double quotes here, or is that a code style error?

File: web/server.js [hunk 4]
```diff
@@ -32,7 +32,41 @@ module.exports = function(options) {
...
+          '\'none\''
+        ],
+        scriptSrc: [
+          '\'self\'',
+          '\'unsafe-eval\'',
+          'https://www.google-analytics.com',
+          'http://www.google-analytics.com'
+        ],
+        fontSrc: [
+        '\'self\'',
```
Comment (jbuck) [inline web/server.js]: indent is off here

File: web/server.js (hunk 2 above)
Comment (cadecairos) [inline web/server.js]: well, according to MDN it'll fall back on default-src, which would be 'self'.

will we ever even want iframes?

File: web/server.js (hunk 3 above)
Comment (cadecairos) [inline web/server.js]: JSCS FTL

File: web/server.js (hunk 4 above)
Comment (cadecairos) [inline web/server.js]: look again

File: web/server.js (hunk 1 above)
Comment (jbuck) [inline web/server.js]: fml

File: web/server.js (hunk 2 above)
Comment (jbuck) [inline web/server.js]: Lets set default-src to none perhaps?

**Pull request merged**